src/Compilers/CSharp/Test/Semantic/Semantics/FunctionPointerTests.cs

```diff
@@ -1918,5 +1918,34 @@ void M(delegate*<void> ptr)
                 Diagnostic(ErrorCode.ERR_NoSuchMemberOrExtension, "ToString").WithArguments("delegate*<void>", "ToString").WithLocation(6, 13)
             );
         }
+
+        [Fact]
+        public void ArglistCannotBeUsedWithFunctionPointers()
```

Comment:
>ArglistCannotBeUsedWithFunctionPointers [](start = 20, length = 39)

It looks like it is possible to specify VARARG in metadata. Please add a test for consuming metadata like that.

Reply:
Added to `CodeGenFunctionPointersTests.UnsupportedCallingConventionInMetadata`

---
In reply to: [409838193](https://github.com/dotnet/roslyn/pull/43355#discussion_r409838193) [](ancestors = 409838193)